tests/end2end/features/utilcmds.feature

```diff
@@ -164,3 +164,7 @@ Feature: Miscellaneous utility commands exposed to the user.
     Scenario: Using debug-log-filter with invalid filter
         When I run :debug-log-filter blah
         Then the error "filters: Invalid value blah - expected one of: statusbar, *" should be shown
+
+    Scenario: Using debug-log-filter
+        When I run :debug-log-filter webview
+        Then no crash should happen
```

Comment:
Oh, I did't realize that. I need to think about a way to do this (this is one of those cases where you need tests for your tests >_<).